txp/Pos/Txp/Arbitrary.hs

```diff
@@ -118,9 +141,9 @@ buildProperTx triplesList (inCoin, outCoin) = fmap newTx txList
 -- | Well-formed transaction 'Tx'.
 newtype GoodTx = GoodTx
     { getGoodTx :: NonEmpty ((Tx, TxDistribution), TxIn, TxOutAux, TxInWitness)
-    } deriving (Show)
+    } deriving (Generic, Show)
 
-newtype SmallGoodTx = SmallGoodTx { getSmallGoodTx :: GoodTx } deriving (Show)
+newtype SmallGoodTx = SmallGoodTx { getSmallGoodTx :: GoodTx } deriving (Generic, Show)
```

Comment:
Btw, I think we should get rid of these `Small…` types and have `newtype Small a = Small {getSmall :: a}`.